tests/polkadotjs_test/test/test-polkadot.js

```diff
@@ -180,5 +180,11 @@ describe('Testing polkadot.js/api calls:', function () {
             expect(transfer).to.have.lengthOf(32);
         });
     });
-
+    describe('api grandpa', () => {
+        it('call api.rpc.grandpa.proveFinality', async function () {
+            const proveBlockNumber = 0;
+            const finality = await api.rpc.grandpa.proveFinality(proveBlockNumber);
+            expect(finality).to.be.not.null;
```

Comment:
😄 Javascript at its best `to.be.not.null`

On a more serious note, should we assert other stuff? Maybe that it matches a regex of some sort?

Reply:
It seems that the result for this is an empty object, so I'm not sure what else to test for.